src/content/developers/docs/nodes-and-clients/client-diversity/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Client diversity
+description: A high level explanation of the importance of Ethereum client diversity.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+
+The behaviour of an Ethereum node is controlled by the client software it runs. There are several production-level Ethereum clients, each one developed and maintained in different languages by separate teams. The clients are built to a common spec that ensures the clients seamlessly communicate with each other and have the same functionality and provide an equivalent user experience. However, at the moment the distribution of clients across nodes is not even enough to realise this network fortification to its full potential. Ideally, users divide roughly equally across the various clients to bring as much client diversity as possible to the network.
+
+## Prerequisites {#prerequisites}
+
+If you don't already have an understanding of what nodes and clients are, check out [Nodes and clients](/developers/docs/nodes-and-clients/). The Beacon Chain is explained [here](/src/content/eth2/beacon-chain/index.md). Execution and consensus layers are defined in the [glossary](/src/content/glossary/index.md).
```

Comment:
These terms are now added, so I used their direct links instead. 
(Btw, links should just be the URL path after the language, so for example `https://ethereum.org/en/glossary/` can just be shortened to `/glossary/`... don't want to include the /src/content/ though)

```suggestion
If you don't already have an understanding of what nodes and clients are, check out [nodes and clients](/developers/docs/nodes-and-clients/). The Beacon Chain is explained [here](/upgrades/beacon-chain/). [Execution](/glossary/#execution-layer) and [consensus]/glossary/#consensus-layer) layers are defined in the glossary.
```